habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -879,7 +872,7 @@ class ThirdDepthSensorConfig(HabitatSimDepthSensorConfig):
 class AgentConfig(HabitatBaseConfig):
     height: float = 1.5
     radius: float = 0.1
-    sensors: List[str] = field(default_factory=lambda: ["rgb_sensor"])
+    camera_sensors: Dict[str, Dict[str, Any]] = field(default_factory=dict)
```

Comment:
Temporary. I wanted to do `Dict[str, SensorConfig]` but this does not work as some keys are present in some SensorConfig implementations but not in the base SensorConfig. Will need to figure out a workaround.

Reply:
This is is a typing issue /architecture issue. @vincentpierre the solution here is to specify a Union across all the ivalid implementations of Sensor Ccnfigs or to redo the hierarchy to add abstract methods the support it as some intermediate ancestor / mixin

Reply:
@Skylion007 This is no longer the Issue. The highlighted code is outdated. Everything works fine with `Dict[str, SensorConfig]`.